### ISSUE TITLE
Debug console autocompletion not working

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/ptvsd#1000]
_From @darrickyee on November 9, 2018 16:21_

## Environment data

- VS Code version: 1.28.2
- Extension version (available under the Extensions sidebar): 2018.10.0
- OS and version: Windows 7
- Python version (& distribution if applicable, e.g. Anaconda): 3.7.0
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A

## Actual behavior

Autocompletion in the debug console console doesn't seem to work on anything except the first name typed.

Autocomplete works when an object is the first thing typed:
![debug1](https://user-images.githubusercontent.com/18172544/48274399-3768eb80-e411-11e8-8de2-efb22f9b8fc1.jpg)

No autocompletion if the same object is on the other side of an assignment, passed as a parameter, or even just enclosed in brackets:
![debug2](https://user-images.githubusercontent.com/18172544/48274484-6a12e400-e411-11e8-9559-50606887aba0.jpg)

Is this the intended behavior?

_Copied from original issue: Microsoft/vscode-python#3282_

**Comment by karthiknadig** [comment]
I don't think this is a regression. The completions were always pretty simple (i.e., the first case). I don't think it ever handled the second case. 
/cc @fabioz

**Comment by fabioz** [comment]
I'll investigate it here (i.e.: it seems to work for me in plain pydev/pydevd, so, it may be some integration issue).

**Comment by fabioz** [comment]
The problem is that the completion command only expects the token for the completion and not the full text entered (that part was expected to be handled by the IDE)... 

I'm in the process of changing the related message to use the DAP json format so that it has the full info on the text/column so that it can do what's needed properly. 

As a note, I'm doing it on top of the changes in the pull request: https://github.com/Microsoft/ptvsd/pull/1070 (so, please don't rebase when merging).
